src/prommis/uky/uky_flowsheet.py

```diff
@@ -10,10 +10,138 @@
+In the rougher circuit, solutes in the aqueous phase are transferred to the organic phase, and a portion of the
+depleted aqueous solution is recycled back to the leaching process while the remainder is sent to the cleaner circuit.
+The degree to which components are transferred from one phase to the other is dependent upon the unit's partition coefficient for that particular component.
+In the cleaner circuit, solutes in the organic phase are transferred to the aqueous phase, and a portion of the loaded
+aqueous solution is recycled back to the rougher circuit while the remainder is sent to the precipitator. The precipitate
+(with some retained liquid) is sent to the roaster where the product rare earth oxides are generated, and the liquid from
+the precipitator is recycled back to the cleaner circuit.
+
+.. figure:: ../tutorials/uky_flowsheet.png
+    :width: 800
+    :align: center
+
+    University of Kentucky flowsheet
+
+Degrees of Freedom
+------------------
+The following variables must be specified by the user to run the UKy flowsheet:
+    * liquid feed volumetric flow rate and component concentrations
+    * solid feed mass flow and component mass fractions
+    * volume of leach tank(s)
+    * partition coefficients for each solvent extraction unit
+    * flow rate and component concentrations for organic make-up streams and HCl feeds
+    * liquid recovery fraction for solid-liquid separators
+    * precipitator inlet temperature
+    * roaster inlet temperature and pressure, outlet temperature, and pressure drop
+    * roaster gas and moisture molar flows, vapor component mole fractions, and oxide recovery fraction
+    * split fractions for each recycle loop
+
+Default Flowsheet Specifications
+--------------------------------
+
+===================================================================== ============ ============================
+Description                                                           Value        Units
+===================================================================== ============ ============================
+Leaching
+Tank volume                                                           100          :math:`\text{m}^3`
+Liquid feed volumetric flow                                           224.3        :math:`\text{L/hr}`
+Liquid feed H concentration                                           100          :math:`\text{mg/L}`
+Liquid feed HSO4 concentration                                        1e-8         :math:`\text{mg/L}`
+Liquid feed SO4 concentration                                         4800         :math:`\text{mg/L}`
+Liquid feed REE and contaminant concentrations                        1e-10        :math:`\text{mg/L}`
+Solid feed mass flow                                                  22.68        :math:`\text{kg/hr}`
+Solid feed inerts mass fraction                                       0.6952       :math:`\text{dimensionless}`
```

Comment:
Are the feed mass fractions correct? I see that many are the same, and it looks like they would not sum to 1.

Reply:
Good catch - I need to correct this.